pyproject.toml

```diff
@@ -27,15 +30,12 @@ flake8 = "^6.0.0"
 flake8-tuple = "^0.4.1"
 flake8-bugbear = "^23.1.20"
 mypy = "^0.991"
-eth-account = "^0.5.6"
-eth-brownie = "^1.19.0"
 pylint = "^2.16.1"
 types-pkg-resources = "^0.1.3"
 types-requests = "^2.28.11"
 pdbpp = "^0.10.3"
 pytest-cov = "^4.0.0"
 shiv = "^1.0.3"
-Sphinx = "^6.1.3"
```

Comment:
It is strange that we are removing sphinx when we need it for our docs. What was the reason for this?

Reply:
i updated ape to use sphinx 6.1.3 and lock should have like that but i will add it here also